README.md

```diff
@@ -2,7 +2,6 @@
 
 [![made-with-python](https://img.shields.io/badge/Made%20with-Python%203.8+-ffe900.svg?longCache=true&style=flat-square&colorB=00a1ff&logo=python&logoColor=88889e)](https://www.python.org/)
 [![AGPL](https://img.shields.io/badge/Licensed%20under-AGPL-red.svg?style=flat-square)](./LICENSE)
-[![Validation](https://github.com/Scoder12/rickchurch/actions/workflows/validation.yaml/badge.svg)](https://github.com/Scoder12/rickchurch/actions/workflows/validation.yaml)
```

Comment:
It was done for the old repo, I forgot to update it, it should just be a matter of changing the url to this one, workflow name is the same

Reply:
I already tried to change just the GitHub username, and the badge still wasn't appearing. Ill double check that though!

Reply:
If not, just go to actions, find validation workflow and click create status badge, but it should work just with the url change

Reply:
Yep, it just worked! I just made a new commit containing the fixed version!